File: september/24-09/24-09.js
function isPerfectSquare(n) {

    let isPerfect = false;
    let square = Math.sqrt(n);
    //console.log(square);

    if(n >= 0){
        if(square % 1 == 0){
            isPerfect = true;
        }else{
            isPerfect = false;
        }
    }else{
        isPerfect = false;
        
    }

    console.log(isPerfect);

    return isPerfect;
}

File: september/24-09/24-09.java
boolean isPerfectSquare(int n){

        double square = Math.sqrt(n);

        if(square >= 0){
            
            if(square % 1 == 0){
                return true;
            
            }else{
                
                return false;
            }
        }else{

            return false;
        }

}    
    

    

